source/faq.rst

```diff
@@ -0,0 +1,243 @@
+
+For more information, run
+
+.. code-block:: bash
+
+   $ pmm-admin add mysql --help
+
+Can I rename instances?
+================================================================================
+
+You can remove any monitoring instance as described in :ref:`pmm-admin.rm`
+and then add it back with a different name.
+
+When you remove a monitoring service, previously collected data remains
+available in |grafana|.  However, the metrics are tied to the instance name.  So
+if you add the same instance back with a different name, it will be considered a
+new instance with a new set of metrics.  So if you are re-adding an instance and
+want to keep its previous data, add it with the same name.
+
+.. _troubleshoot-connection:
+
+How to troubleshoot communication issues between PMM Client and PMM Server?
+================================================================================
+
+Broken network connectivity may be caused by rather wide set of reasons.
+Particularly, when :ref:`using Docker <run-server-docker>`, the container is
+constrained by the host-level routing and firewall rules. For example, your
+hosting provider might have default *iptables* rules on their hosts that block
+communication between |pmm-server| and |pmm-client|, resulting in *DOWN* targets
+in Prometheus. If this happens, check firewall and routing settings on the
+Docker host.
+
+Also |pmm| is able to generate a set of diagnostics data which can be examined
+and/or shared with Percona Support to solve an issue faster. You can get
+collected logs from PMM Client using the ``pmm-admin summary`` command. 
+Obtaining logs from PMM Server can be done `by specifying the
+``https://<address-of-your-pmm-server>/managed/logs.zip`` URL, or by clicking
+the ``server logs`` link on the `Prometheus dashboard <https://www.percona.com/doc/percona-monitoring-and-management/2.x/dashboards/dashboard-prometheus.html>`_:
+
+.. image:: .res/graphics/png/get-logs-from-prometheus-dashboard.png
+
+.. only:: showhidden
+
+	.. _metrics-resolution:
+
+	What resolution is used for metrics?
+	================================================================================
+
+	The |opt.mysql-metrics| service collects metrics with different resolutions (1
+	second, 5 seconds, and 60 seconds)
+
+	The |opt.linux-metrics| and |opt.mongodb-metrics| services are set up to collect
+	metrics with 1 second resolution.
+
+	In case of bad network connectivity between |pmm-server| and |pmm-client| or
+	between |pmm-client| and the database server it is monitoring, scraping every
+	second may not be possible when latency is higher than 1 second.  You can change
+	the minimum resolution for metrics by passing the ``METRICS_RESOLUTION``
+	environment variable when :ref:`creating and running the PMM Server container
+	<server-container>`.  To set this environment variable, use the ``-e`` option.
+	The values can be between *1s* (default) and *5s*.  If you set a higher value,
+	|prometheus| will not start.
```

Comment:
see Prod. Req about 3 resolutions. there are different metrics in different resolutions and default is 5s-5s-60s for now

Reply:
This is a hidden section, which waits for the settings gui which definitely should be referenced here in the 1st place.